samples/bluetooth/tmap_central/src/cap_initiator.c

```diff
@@ -0,0 +1,513 @@
+/*
+ * Copyright (c) 2022-2023 Nordic Semiconductor ASA
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if defined(CONFIG_BT_CAP_INITIATOR)
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <zephyr/kernel.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/bap_lc3_preset.h>
+#include <zephyr/bluetooth/audio/cap.h>
+#include <zephyr/bluetooth/audio/bap.h>
+#include "common.h"
+
+static struct k_work_delayable audio_send_work;
+
+static struct bt_cap_stream unicast_streams[CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT +
+										CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT];
```

Comment:
Some indentation went wrong here. Please note that tabs in Zephyr are 8 spaces :)

Reply:
This seems to be a general issue in this commit :)